$OpenBSD: patch-vector_cabal,v 1.1 2016/12/10 21:20:45 kili Exp $

Dependencies changed in the "online" version of the package description,
but not in the dist file.

--- vector.cabal.orig	Mon Jul 13 06:26:16 2015
+++ vector.cabal	Wed Nov 30 21:09:18 2016
@@ -137,9 +137,9 @@ Library
   Install-Includes:
         vector.h
 
-  Build-Depends: base >= 4.3 && < 4.9
-               , primitive >= 0.5.0.1 && < 0.7
-               , ghc-prim >= 0.2 && < 0.5
+  Build-Depends: base >= 4.3 && < 4.10
+               , primitive >= 0.5.0.1 && < 0.6.2
+               , ghc-prim >= 0.2 && < 0.6
                , deepseq >= 1.1 && < 1.5
 
   Ghc-Options: -O2 -Wall -fno-warn-orphans
@@ -189,7 +189,7 @@ test-suite vector-tests-O2
   hs-source-dirs: tests
   Build-Depends: base >= 4 && < 5, template-haskell, vector,
                  random,
-                 QuickCheck  >= 2.7, test-framework, test-framework-quickcheck2,
+                 QuickCheck  >= 2.7 && < 2.8 , test-framework, test-framework-quickcheck2,
                  transformers >= 0.2.0.0
 
   default-extensions: CPP,
